pipelines/fbc-release/fbc-release.yaml

```diff
@@ -4,7 +4,7 @@ kind: Pipeline
 metadata:
   name: fbc-release
   labels:
-    app.kubernetes.io/version: "0.26.0"
+    app.kubernetes.io/version: "0.27.0"
```

Comment:
I've been going to `1.0.0` for these changes. Not 100% sure when this is appropriate, but if this change isn't one that bumps the major version, I don't know what is